app/views/layouts/_header.html.haml

```diff
@@ -15,7 +15,7 @@
         %li
           %a{:href => "#" } About
         %li
-          %a{:href => "#" } Contact
+          %a{:href => conversations_path } Conversations
```

Comment:
give preference to helper 
should be like
= link_to 'Conversations', conversations_path